src/assets/themes/apollo-notype.json

```diff
@@ -26,5 +26,5 @@
         "pin": "#ebebeb",
         "satellite": "#ebebeb"
     },
-    "injectCSS": "section#filesystem{left:0;width:100vw}section#filesystem>h3.title,section#filesystem>div{width:100vw}section#keyboard{display:none;}"
+    "injectCSS": "section#filesystem{left: 0;width:82vw} section#filesystem>h3.title, section#filesystem>div{width:82vw;padding-left:10px;} section#keyboard{display:none;}"
```

Comment:
Same matter as above... Does this break on 16:9 and other common screen ratios?

Reply:
I tested this code in multiple aspects ratios and it works as expected. But I will follow your advice, move my changes to the recommended location and make sure it works.